### PR TITLE
First 3d game tutorial documentation fix for 4.1.3 engine version

### PR DESCRIPTION
**Pull request by yvd**
Step missing for testing the player movement in [getting_started/first_3d_game/03.player_movement_code](https://docs.godotengine.org/en/stable/getting_started/first_3d_game/03.player_movement_code.html) tutorial.

### CODE REVIEW

File: getting_started/first_3d_game/03.player_movement_code.rst [hunk 1]
```diff
@@ -369,6 +367,11 @@ Child Scene*.
 In the popup, double-click ``player.tscn``. The character should appear in the
 center of the viewport.
 
+Select the newly instanciated Player node, in the Inspector tab under Node 
+section, load the previously created player script i.e. Player.gd.
```
Comment (AThousandShips) [inline getting_started/first_3d_game/03.player_movement_code.rst:371]:
```suggestion
section, load the previously created player script i.e. ``Player.gd``.
```

Comment (AThousandShips) [reply getting_started/first_3d_game/03.player_movement_code.rst:371]: Though this is not correct, the script is already attached, as per:
> Right-click the ``Player`` node and select *Attach Script* to add a new script to
it. In the popup, set the *Template* to *Empty* before pressing the *Create*
button.

So you must have missed this instruction if you feel this is needed to be added here?

**Review by AThousandShips — CHANGES_REQUESTED** [review]

File: getting_started/first_3d_game/03.player_movement_code.rst (hunk 1 above)
Comment (yvd) [reply getting_started/first_3d_game/03.player_movement_code.rst:371]: The player script was already attached in the player scene. But after instancing the player in the main scene and running the main scene, the controls were not working. That's when I saw the script was empty in the Inspector when selecting the player node in the main scene. After attaching it there, I was able to control the character in the main scene.

**Comment by yvd** [comment]
I followed the tutorial completely as per my knowledge and did not miss the part @AThousandShips has mentioned. While importing the project in the first step, [Setting up the game area](https://docs.godotengine.org/en/stable/getting_started/first_3d_game/01.game_setup.html#setting-up-the-game-area), I used the auto convert of the project from 3.5 to 4.1. Other than that I followed the tutorial completely. @skyace65

**Comment by AThousandShips** [comment]
But the scene is set up *before* converting it, so there's no effect from that part, did you save the scene correctly before instancing it? This could be related to some bug as well so please check by creating the project again and look at bug reports [here](https://github.com/godotengine/godot/issues) 🙂

**Comment by yvd** [comment]
Most probably a bug since I did not have to do this step while following the 2d game tutorial. After Instantiating Player node the scene looks as follows:

And this is not working. After attaching script on the Player Node, it is working.

Will check the bug list and create one if required, thanks! :)

**Pull request closed without merging**